src/Timeline/TimelineDataBuilder.php

```diff
@@ -89,6 +89,15 @@ private function getExecutors(): array
         return $executors;
     }
 
+    private function buildItemTitle(\SimpleXMLElement $testElement): string
+    {
+        return sprintf(
+            '%s<br>(%d sec)',
+            $this->assembleFullTestName($testElement),
+            $this->calculateTestDurationSeconds((string) $testElement['start'], (string) $testElement['end'])
```

Comment:
what if the value is not set?

It would throw `notice` and then it would send an empty string to `calculateTestDurationSeconds` which will create `now` instead of `start|end` date.

Reply:
`$testElement` is an instance of SimpleXMLEelement, and as a such, it don't throw notices on accessing empty attributes, but returns null for them. Typacasting then turn the null to empty string - and this is expected behavior (considering _now_ as the time if not defined). Because it could only means the test has not yet finished, it. it should be shown on the timeline as running until "now".